server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileController.java

```diff
@@ -76,11 +76,13 @@ public MemberProfileController(MemberProfileServices memberProfileServices){
      * @param name
```

Comment:
we probably want to include workEmail on line 75

Reply:
good catch done